.env.dev

```diff
@@ -18,7 +18,5 @@ REACT_APP_THROW_SAVE_PROFILE_ERRORS=true
 REACT_APP_WEB3_SITE_URL=https://w3.gooddollar.org
 REACT_APP_WEB3_SITE_URL_ECONOMY_ENDPOINT=/learn/economy
 REACT_APP_MNEMONIC_TO_SEED=true
-REACT_APP_MARKET_URL=https://etoro.paperclip.co
```

Comment:
keep this env
with default https://www.facebook.com/groups/gooddollarmarketplace
change this to default every where also in config.js